spec.md

```diff
@@ -413,6 +419,22 @@ message CreateVolumeResponse {
   }
 }
 
+message Principal {
+  // describes attributes for defining the principal identity of 
+  // a volume.
+
+  // This field is OPTIONAL. If specified it contains the CO namespace
+  // identity in which the the following identity information (user /
+  // group) exists.
+  string namespace = 1;
```

Comment:
> The design posits that a container acts as one specific Principal when accessing storage

I think this assumption is pretty limiting. I'd like to see something a bit more flexible.